packages/api/test/analytics-test.js

```diff
@@ -59,7 +59,7 @@ const testCases = [
 
 describe('Flaky-Analytics', () => {
   before(async () => {
-    global.headCollection = 'testing/' + 'analytics-testsuite-' + uuidv4() + '/repos'; // random collection name for concurrent testing
+    global.headCollection = 'testing/' + Date.now() + '-analytics-testsuite-' + uuidv4() + '/repos'; // random collection name for concurrent testing
```

Comment:
I added a `timestamp` as a prefix to our collections, so that we can use it to decide if data is stale when deleting.

Reply:
Makes sense! I can't remember, but we track the timestamp for individual tests' runs as well, right? I'd thought about checking the runtime on one of the tests to decide if data is stale, but this seems much more straightforward.